### DISCUSSION TITLE
Maven Implementation-Version

### DISCUSSION THREAD

**Discussion by tainn** [runelite/runelite#15497] [Ideas]
Hello.

I use the `jar` archive of RuneLite, and it would be helpful if I could fetch the version from the `META-INF/MANIFEST.MF` for my usability purposes.

To add the Implementation-Version, the `maven-jar-plugin` inside the root `pom.xml` has to be adjusted to include the following configuration:

```xml
<configuration>
    <archive>
        <manifest>
            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
        </manifest>
    </archive>
</configuration>
```

It's an addition that adds some useful reflection without additional overhead or complexity, outside of the configuration block. Due to its simplicity, I could open up a PR for it, but `CONTRIBUTING.md` prompts for opening up a discussion first.

Thank you.

**Comment by LlemonDuck** [comment]
You can already pull the version from `META-INF/maven/net.runelite/client/pom.xml` if that is a viable alternative for your use case
